stglib/vec/dat2cdf.py

```diff
@@ -206,8 +225,29 @@ def load_dat(basefile):
         "AnalogInput2",
         "Checksum",
     ]
+
     dat = pd.read_csv(f"{basefile}.dat", header=None, sep=r"\s+", names=names)
-    dat = dat.set_index(["Burst", "Ensemble"])
+
+    if metadata["instmeta"]["VECBurstInterval"] != "CONTINUOUS":
+        array = np.zeros(np.shape(dat.VEL1))
+
+        for i in range(np.shape(dsvhd["time"])[0]):
```

Comment:
look into this

Reply:
Switching from using dsvhd["time"] shape to using the vhd burst values. I think that's a better way to create time. I've written it so the code will use Burst number from the vhd and dat files in order to find the corresponding burst start time. After, I use the ensemble (sample) number along with frequency to increase time throughout the burst. This way, it can account for any possible jumps between samples.